easybuild/easyconfigs/e/EnsEMBLCoreAPI/EnsEMBLCoreAPI-96.0-r20190601-foss-2019a-Perl-5.28.1.eb

```diff
@@ -0,0 +1,340 @@
+    }),
+    ('Sereal::Encoder', '4.007', {
+        'source_tmpl': 'Sereal-Encoder-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/Y/YV/YVES'],
+        'checksums': ['bf6fdddc8fcc901c78adcfb61f56c393cd64d73ab320195ebae9e4a82976eab6'],
+    }),
+    ('Socket', '2.029', {
+        'source_tmpl': '%(name)s-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/P/PE/PEVANS'],
+        'checksums': ['3a8d1710a28bf5d7bc46737e43e83657ed4b1c1842efc4cde06c9dd3f6831f06'],
+    }),
+    ('Storable', '3.15', {
+        'source_tmpl': '%(name)s-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/X/XS/XSAWYERX'],
+        'checksums': ['fc3dad06cb2e6fc86a2f2abc5b5491d9da328ca3e6b6306559c224521db174da'],
+    }),
+    ('String::Approx', '3.28', {
+        'source_tmpl': 'String-Approx-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/J/JH/JHI'],
+        'checksums': ['43201e762d8699cb0ac2c0764a5454bdc2306c0771014d6c8fba821480631342'],
+    }),
+    ('Test', '1.26', {
+        'source_tmpl': '%(name)s-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/J/JE/JESSE'],
+        'checksums': ['f7701bd28e05e7f82fe9a181bbab38f53fa6aeae48d2a810da74d1b981d4f392'],
+    }),
+    ('Test::More', '1.302162', {
+        'source_tmpl': 'Test-Simple-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/E/EX/EXODIST'],
+        'checksums': ['73a0197c3d735e060881da509f4c5fa76c4cbf7dbabd4297d3cf5f5feed31fc4'],
+    }),
+    ('Text::ParseWords', '3.30', {
+        'source_tmpl': 'Text-ParseWords-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/C/CH/CHORNY'],
+        'checksums': ['85e0238179dd43997e58c66bd51611182bc7d533505029a2db0d3232edaff5e8'],
+    }),
+    ('Text::Wrap', '2013.0523', {
+        'source_tmpl': 'Text-Tabs+Wrap-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/M/MU/MUIR/modules'],
+        'checksums': ['b9cb056fffb737b9c12862099b952bf4ab4b1f599fd34935356ae57dab6f655f'],
+    }),
+    ('Time::Local', '1.28', {
+        'source_tmpl': 'Time-Local-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/D/DR/DROLSKY'],
+        'checksums': ['9278b9e5cc99dcbb0fd27a43e914828b59685601edae082889b5ee7266afe10e'],
+    }),
+    ('URI::Escape', '3.31', {
+        'source_tmpl': 'URI-1.76.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/O/OA/OALDERS'],
+        'checksums': ['b2c98e1d50d6f572483ee538a6f4ccc8d9185f91f0073fd8af7390898254413e'],
+    }),
+    ('XML::Writer', '0.625', {
+        'source_tmpl': 'XML-Writer-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/J/JO/JOSEPHW'],
+        'checksums': ['e080522c6ce050397af482665f3965a93c5d16f5e81d93f6e2fe98084ed15fbe'],
+    }),
+    ('XSLoader', '0.24', {
```

Comment:
@lexming Any reason not to include these extensions in the `Perl` easyconfig instead?

Reply:
@boegel the only reason is that these extensions are currently only used by this easyconfig afaik. But apart from this nothing hinders their inclusion in `Perl` instead. What do you prefer?

Reply:
@boegel revisiting this after more than 4 months. I moved `Set::InetrvalTree` and `URI::Escape` (and their ~40 dependencies) to main Perl easyconfig as these are used by other stuff (`STAR-Fusion`). The others are quite specific and some require non-Perl dependencies, so I'm leaving those in this easyconfig.